demo/src/main/java/com/example/demo/WriteToFile.java

```diff
@@ -0,0 +1,46 @@
+package com.example.demo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+
+public class WriteToFile {
```

Comment:
Make this a spring component instead of static accessors. This is the example of high coupling that we're trying to solve using spring